appeals/web/src/server/lib/mappers/appeal/submappers/site-visit-date.mapper.js

```diff
@@ -13,9 +12,9 @@ export const mapSiteVisitDate = ({ appealDetails, currentRoute }) => ({
 			value: {
 				html:
 					dateAndTimeFormatter(
-						dateToDisplayDate(appealDetails.siteVisit?.visitDate),
-						convert24hTo12hTimeStringFormat(appealDetails.siteVisit?.visitStartTime),
-						convert24hTo12hTimeStringFormat(appealDetails.siteVisit?.visitEndTime)
+						dateISOStringToDisplayDate(appealDetails.siteVisit?.visitDate),
+						dateISOStringToDisplayTime12hr(appealDetails.siteVisit?.visitStartTime),
+						dateISOStringToDisplayTime12hr(appealDetails.siteVisit?.visitEndTime)
```

Comment:
Again we can tackle separately, but might make sense for `dateAndTimeFormatter` to accept three dates, and convert for display in that function.